zlib-ng.h

```diff
@@ -522,7 +528,8 @@ ZEXTERN int ZEXPORT zng_inflateEnd(zng_stream *strm);
 */
 
 /*
-ZEXTERN int ZEXPORT zng_deflateInit2 (zng_stream *strm,
+ZEXTERN ZEXPORT
+int zng_deflateInit2(zng_stream *strm,
                                      int  level,
```

Comment:
Can you fix this in the comments? The function parameters span multiple lines.

Reply:
Done